typescript/cli/src/utils/balances.ts

```diff
@@ -8,9 +8,10 @@ export async function assertNativeBalances(
   signer: ethers.Signer,
   chains: ChainName[],
   minBalanceWei: string,
-) {
+): Promise<boolean> {
```

Comment:
i think things named assert i expect to throw usually, vs this change makes the function now return a bool. and i have to think a bit whether lowBalanceExists means the balance is too low, or it's not - vs something like `balanceIsSufficient` or something like that i just find easier to reason (true - balance is sufficient, false -  balance is not enough)

Reply:
Completely agree– changing the return and not refactoring the name was poor retrofitting.

Updated to `nativeBalancesAreSufficient` where `sufficient` represents if a low balance is found or not